pkg/operator/controller/ingress/controller_test.go

```diff
@@ -652,20 +652,6 @@ func TestSetDefaultPublishingStrategyHandlesUpdates(t *testing.T) {
 			expectedIC:              makeIC(spec(elb()), status(elbWithNullParameters())),
 			domainMatchesBaseDomain: true,
 		},
-		{
-			name:                    "loadbalancer type changed from ELB to NLB, with old ELB parameters removal",
-			ic:                      makeIC(spec(nlb()), status(elbWithNullParameters())),
-			expectedResult:          true,
-			expectedIC:              makeIC(spec(nlb()), status(nlbWithNullParameters())),
-			domainMatchesBaseDomain: true,
-		},
-		{
-			name:                    "loadbalancer type changed from NLB to ELB, with old NLB parameters removal",
-			ic:                      makeIC(spec(elb()), status(nlbWithNullParameters())),
-			expectedResult:          true,
-			expectedIC:              makeIC(spec(elb()), status(elbWithNullParameters())),
-			domainMatchesBaseDomain: true,
-		},
```

Comment:
Is there a reason you removed these test cases rather than changing them to match the new behavior?  Granted, `TestSetDefaultPublishingStrategyHandlesUpdates` is already a bit unwieldy, and you did add plenty of test coverage in this commit, and that should be adequate.